Dockerfile

```diff
@@ -0,0 +1,24 @@
+# This is a Dockerfile that extends an installation of itk and modules
+#in the dsarchive/histomicstk:v0.1.3
+
+
+FROM dsarchive/histomicstk:v0.1.3
+
+MAINTAINER Bilal Salam <bilal.salam@kitware.com>
+#install requirements of pygco
+RUN cd / && \
+    git clone https://github.com/yujiali/pygco.git && \
+    cd pygco && \
+    make && \
+    echo 'export PYTHONPATH=$PYTHONPATH:/pygco' >> ~/.bashrc 
+
+
+ENV my_plugin_path=$htk_path/../my_plugin
+RUN mkdir -p $my_plugin_path
+COPY . $my_plugin_path
+WORKDIR $my_plugin_path
+
+
+# use entrypoint provided by HistomicsTK
```

Comment:
Before invoking `ENTRYPOINT` we need to set WORKDIR to the directory that contains slicer_cli_list.json which is located in the [Applications](https://github.com/cooperlab/NucleiCuts/tree/master/Applications) in this repository.

So put the following command above the `ENTRYPOINT` command

```Dockerfile
WORKDIR $my_plugin_path/Applications
```

Also test the following:
* `docker run <image-name> --list_cli
* `docker run <image-name> <cli-name> --xml